KyberElasticAdapter/contracts/interfaces/IWETH.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./IERC20.sol";
+
+interface IWETH is IERC20 {
+    function withdraw(uint256 amount) external;
+
+    function deposit() external payable;
```

Comment:
Missing NatSpec
Add descriptions for the interface, events and functions
Refer https://docs.soliditylang.org/en/v0.8.17/natspec-format.html